### ISSUE TITLE
Attract.exe won't boot - Windows 10 - Solved??

### ISSUE THREAD

**Issue by ThisorThatguy** [mickelson/attract#654]
Not sure if this is known or not. I did some research and didn't find anything so I wanted to share my fix.

I had a few issues with attract mode where it would launch fine I would make some edits to displays or something and when i would try again it would fail to load at all. How I would fix this was by pasting a backup of the main Attract folder files (IE attract.exe, attract.cfg, attract.am.... ect).

in last_run.log i would get "Error, could not find default font."

After doing some more digging after the last fail to launch happened i discovered the cause was attract.am. After a little more research I found that on line 1 of attract.am you will have something that looks like this( 8;107,0,4; ) If the the first spot (the 8 in the example) is either 0 or 1 Attract.exe will not launch. 

When I put 0 or 1 in that first spot on line 1 I got the font error everytime. I tried 2-10 without issues so if you get that error just change that first value and give it a go!

I hope this helps someone and if this has already been posted sorry to double post on it 😁

**Comment by oomek** [comment]
The variables stored in attract.am file are the following:
display_idx
m_last_launch_display
m_last_launch_filter
m_last_launch_rom
m_menu_layout_file
Would you paste a debug log please when it crashes?

**Comment by ThisorThatguy** [comment]
Sorry forgot to mention a few things this was on windows 10 (not sure if that matters) and i am using HyperPC. I just started to mess around with AM a week or so ago where would I go to pull the debug log? I can force the fail anytime by changing the value. I tried a few more experiments and on my backup setup if I change the settings that one will work so it seems like something else might be at play. It is still only fails if i change it to a 1 or 0 (doesn't like the binary! haha)

I tried moving my original displays files over but still 0 and 1 cause the issue. I redownloaded hyperPC.7z and pulled the files from there and one by one tested the files and it is my attract.cfg that causes the error.

So finally after all that fun it looks like my default font path was messed up. The error I would get in last run log wass 'could not find default font' (which i thought was a false positive since it only happened on 0 or 1) but I put the roboto font into layouts and changed the font path to match.. and now there is are longer issues with 0 or 1 causing boot fails...

Sorry was typing this out while troubleshootings 😁 if there is anything you think i missed let me know! I appreciate the help!
